aws/data_source_aws_prefix_list_test.go

```diff
@@ -19,6 +20,10 @@ func TestAccDataSourceAwsPrefixList_basic(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					testAccDataSourceAwsPrefixListCheck("data.aws_prefix_list.s3_by_id"),
 					testAccDataSourceAwsPrefixListCheck("data.aws_prefix_list.s3_by_name"),
+					resource.TestMatchResourceAttr("data.aws_prefix_list.s3_by_name", "id", regexp.MustCompile(`^pl-[0-9a-z]{8}$`)),
+					resource.TestCheckResourceAttr("data.aws_prefix_list.s3_by_name", "name", fmt.Sprintf("com.amazonaws.%s.s3", testAccGetRegion())),
```

Comment:
Can use this helper which also accounts for the reverse DNS prefix changing between partitions:

```suggestion
					testAccCheckResourceAttrRegionalReverseDnsService("data.aws_prefix_list.s3_by_name", "name", "s3"),
```